### PR TITLE
[v2] Add support of H3ULCB ES2.0 Cetibox machine

### PR DESCRIPTION
**Pull request by andr2000**
This PR supersedes PR #45

### CODE REVIEW

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-core/systemd/files/eth0.network [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Match]
+Name=eth0
+KernelCommandLine=!nfsroot
+
+[Network]
+VLAN=eth0.1
+VLAN=eth0.2
+
+[Link]
+MACAddress=11:a9:87:65:43:66
```
Comment (andr2000) [inline recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-core/systemd/files/eth0.network]: It is not clear if this MAC address is valid and relevant @al1img

**Comment by andr2000** [comment]
It is not clear if https://github.com/xen-troops/meta-xt-prod-aos/commit/63acecb48c0c2d4f182338cfb3dddcc591a49bb8 is needed.

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-core/systemd/files/eth0.network (hunk 1 above)
Comment (rshym) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-core/systemd/files/eth0.network]: For now we see that board has permanent MAC 02:01:02:03:04:08. And this address is present in /usr/bin/eth_full_reset.sh.
But have to note that attempt to set any other address results in random MAC. So need to clarify.

Comment (rshym) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-core/systemd/files/eth0.network]: Seems like this MACAddress has no effect.
"MACAddress=
The MAC address to use, if no MACAddressPolicy= is specified."

But we have /usr/lib/systemd/network/99-default.link which contains "MACAddressPolicy=persistent".

**Pull request merged**